ehealth-portal-backend/manifests/ehealth-portal-backend-deployment.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ehealth-portal-backend
+  namespace: openncp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ehealth-portal-backend
+  template:
+    metadata:
+      labels:
+        app: ehealth-portal-backend
+    spec:
+      containers:
+      - name: ehealth-portal-backend
+        image: docker.io/3ahealth/ehealth-portal-backend:7.1.0
+        ports:
+        - containerPort: 8080
+        - containerPort: 8443
+        resources:
+          requests:
+            memory: "512Mi"
+            cpu: "500m"
+          limits:
+            memory: "1Gi"
+            cpu: "1"
+        env:
+        - name: EPSOS_PROPS_PATH
+          value: "/opt/epsos-configuration/"
```

Comment:
Please add this to the readme, the volume mount should be on every node that the services will run.

Reply:
Will adjust in a future PR where I will add ALL the details needed in the README! So far I added a lot of information but is not quite the final README.